src/seedu/addressbook/parser/Parser.java

```diff
@@ -50,36 +50,37 @@ public Parser() {}
      * @param userInput full user input string
      * @return the command based on the user input
      */
-    public Command parseCommand(String userInput) {
+    public static Command parseCommand(String userInput) {
```

Comment:
Don't forget to change the code where this method is used. It should be changed to `Parser.parseCommand(...)` in `Main` class and `ParserTest` class.